assemblies/managing-resources.adoc

```diff
@@ -0,0 +1,101 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+:context: managing-resources
+
+[id="managing-resources"]
+= Managing resources
+
+[role='_abstract']
+ifdef::upstream[]
+
+As an {productname-long} administrator, you can manage these resources:
+
+* Cluster resources to support compute-intensive data science work.
+* Jupyter notebook administration.
+* Data storage backup.
+endif::[]
+ifndef::upstream[]
+As a {productname-long} administrator, you can manage these resources:
+
+* Cluster resources to support compute-intensive data science work.
+* Jupyter notebook administration.
+* Data storage backup.
+* Whether to allow Red Hat to collect data about {productname-short} usage in your cluster.
+
+endif::[]
+
+include::modules/allocating-additional-resources-to-openshift-data-science-users.adoc[leveloffset=+1]
```

Comment:
We should probably rename these files, where possible, to eliminate the product name.

More important, though, is the module ID. If that includes the OpenShift Data Science product name, then this will get included in the URL in the upstream docs.

Reply:
fixed